scripts/tslint/enforceConsecutiveFunctionOverloadRule.ts

```diff
@@ -10,6 +10,26 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 }
 
+function isStringOrNumericLiteral(kind: number) {

```

Comment:
Use `ts.SyntaxKind.StringLiteral` etc instead of `9` / `8` and have `kind: ts.SyntaxKind`